waspc/cli/src/Wasp/Cli/Command/Start/Db.hs

```diff
@@ -157,6 +160,30 @@ startPostgreDevDb waspProjectDir appName = do
                   Dev.Postgres.defaultDevPort
               )
 
+    throwIfDockerDaemonIsNotRunning :: Command ()
+    throwIfDockerDaemonIsNotRunning = do
+      whenM (liftIO isDockerStopped) throwDockerDaemonIsNotRunningError
+      where
+        isDockerStopped = do
+          (exitCode, errorString, _) <-
+            -- For docker formatting reference, see https://pkg.go.dev/text/template#hdr-Functions
+            readProcessWithExitCode "docker" ["info", "--format", "{{ index .ServerErrors 0 }}"] ""
+          pure $ exitCode == ExitSuccess && errorString == "Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?\n"
+
+        throwDockerDaemonIsNotRunningError =
+          E.throwError $
+            CommandError
+              "Docker daemon is not running"
+              ( unlines
+                  [ "Wasp can't run dev database for you because docker daemon isn't running",
+                    "in the background. Have you tried executing `systemctl start docker` before running",
```

Comment:
`systemctl` makes sense only for OSes using systemd. Mac doesn't use it if I am correct, and not all Linux OSes use it. So probably best to skip this specific advice and let's just go for those links.